### DISCUSSION TITLE
html lang 如何设置？

### DISCUSSION THREAD

**Discussion by Caladog** [umijs/umi#12355] [Q&A]
默认打包时，<html lang="en"> 我想改成为中文。
以前设置：
  html: {
    lang: 'en',
  },
  现在不好用了。Invalid config keys: html
  目前版本是："umi": "4.1.5"

**Comment by fz6m** [comment]
参考 https://github.com/umijs/umi-next/issues/868#issuecomment-1151088426 ，在里面找到 html 插入 lang 。
